### PR TITLE
Update documentation for Datadog for IntelliJ plugin

### PR DESCRIPTION
**Pull request by jfree**
### What does this PR do?
Updates the documentation for the Datadog plugin for IntelliJ IDEA.

### Motivation
The latest public beta release (v0.7.0) has new features and the UI has changed, so this PR brings the documentation up to date with respect to our current release.

https://datadoghq.atlassian.net/wiki/spaces/IP/pages/2312798790/Datadog+for+IntelliJ+IDEA

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/developers/ide_integrations/idea/_index.md [hunk 1]
```diff
@@ -94,25 +118,19 @@ The **Top List** sub-tab shows the methods that consume the most resources based
...
+The default **Caller Hierarchy** view shows the callers (or predecessors) of the target method and the frequency with which they appear in the call stack. To view the callees (or successors), click the **Callee Hierarchy** button on the toolbar.
 
 Right-click on a method in the call tree to see options to navigate to the source editor or flame graph.
 
-## Flame graph
+### Flame graph
 
-A flame graph is a visualization of profiling samples that shows stack traces and their relative frequency during the sample period. The Datadog plugin aggregates data collected over the requested time frame, and multiple individual profiles are aggregated together. Each individual profile covers a 60 second interval within the requested time frame.
+A flame graph is a visualization of profiling samples that shows stack traces and their relative frequency during the sample period. The Datadog plugin take multiple individual profiles collected over the requested time frame, and aggregates them. Each individual profile covers a 60 second interval within the requested time frame.
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:131]:
```suggestion
A flame graph is a visualization of profiling samples that shows stack traces and their relative frequency during the sample period. The Datadog plugin collects multiple individual profiles from the requested time frame, and aggregates them. Each individual profile covers a 60 second interval within the requested time frame.
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 2]
```diff
@@ -130,19 +148,35 @@ Profiling samples include stack trace and line number information. Use the **Sep
 
 {{< img src="/developers/ide_integrations/idea/separate-flamegraph-by.png" alt="Use the tooltip button to separate frames by method or line number" style="width:40%;" >}}
 
-## Datadog Insights
+### Source Highlighting
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:151]:
```suggestion
### Source highlighting
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 3]
```diff
@@ -130,19 +148,35 @@ Profiling samples include stack trace and line number information. Use the **Sep
...
 
-When a Profiling tab is active, Datadog Insights adds code highlights to the source code editor margin. The Datadog plugin displays an icon in the editor margin and highlights code based on the active Profiling data.
+When the Continuous Profiler tab is active, the plugin adds code highlights to the source code editor margin. For Top Methods an icon appears in the editor margin, and line-level highlights appear in the code based on the active Profiling data.
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:153]:
```suggestion
When the Continuous Profiler tab is active, the plugin adds code highlights to the source code editor margin. For Top Methods, an icon appears in the editor margin, and line-level highlights appear in the code based on the active Profiling data.
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 4]
```diff
@@ -130,19 +148,35 @@ Profiling samples include stack trace and line number information. Use the **Sep
...
 - Hover over the icon to see more information.
 - Click the icon to open the top list Profiling tab or open Profiling in Datadog.
   {{< img src="/developers/ide_integrations/idea/interest-options.png" alt="Click the Datadog icon to open the Profiling data in a tab or in Datadog" style="width:100%;" >}}
 
 The active Profiling tab also affects the IDEA project tree view, which is annotated with the selected profile's metrics:
 {{< img src="/developers/ide_integrations/idea/project-tree-view.png" alt="The project tree annotated with profile metrics from a profile tab" style="width:60%;" >}}
 
+## Logs Navigation
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:161]:
```suggestion
## Logs navigation
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 5]
```diff
@@ -130,19 +148,35 @@ Profiling samples include stack trace and line number information. Use the **Sep
...
+
+You can navigate to the [Log Explorer][5] on the Datadog platform directly from your Java source files. Look out for **View Logs** links following log statements in your source code:
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:163]:
```suggestion
You can navigate to the [Log Explorer][5] on the Datadog platform directly from your Java source files. Look for the**View Logs** links following the log statements in your source code:
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 6]
```diff
@@ -130,19 +148,35 @@ Profiling samples include stack trace and line number information. Use the **Sep
...
+
+{{< img src="/developers/ide_integrations/idea/logs-navigation.png" alt="A source file showing a View Logs link." style="width:100%;" >}}
+
+Clicking the link opens the **Log Explorer** with a query that matches the logger name, log level and log message as closely as possible.
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:167]:
```suggestion
Clicking the link opens the **Log Explorer** with a query that matches the logger name, log level, and log message as closely as possible.
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 7]
```diff
@@ -130,19 +148,35 @@ Profiling samples include stack trace and line number information. Use the **Sep
...
+
+## View In IntelliJ IDEA
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:169]:
```suggestion
## View in IntelliJ IDEA
```

File: content/en/developers/ide_integrations/idea/_index.md [hunk 8]
```diff
@@ -130,19 +148,35 @@ Profiling samples include stack trace and line number information. Use the **Sep
...
+
+The **View in IntelliJ** feature provides a link from the Datadog platform directly to your source files. Look out for the button next to frames in stack traces displayed on the platform (for example, in [Error Tracking][6]):
```
Comment (aliciascott) [inline content/en/developers/ide_integrations/idea/_index.md:171]:
```suggestion
The **View in IntelliJ** feature provides a link from the Datadog platform directly to your source files. Look for the button next to frames in stack traces displayed on the platform (for example, in [Error Tracking][6]):
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Thanks @jfree for the additions! I have added some small changes for sentence case, formatting, punctuation where needed, and minor wording changes. Also please review the "Log Navigation support" line as it doesn't seem to flow where it is, and should be moved down to the "Log Navigation" section? Or is this just a sentence there to highlight the basic overview of Log Navigation ?Thanks!

File: content/en/developers/ide_integrations/idea/_index.md [hunk 9]
```diff
@@ -4,37 +4,51 @@ kind: documentation
...
 
-{{< img src="/developers/ide_integrations/idea/overview1.png" alt="The Datadog tool window open in IDEA" style="width:100%;" >}}
-
 In addition to code highlighting, you can use the Datadog plugin for Intellij IDEA to:
 - Identify methods that consume the most resources with the **Top List** view.
 - Visualize aggregated profile data with the **Flame Graph** view.
 - Navigate directly from the top list and flame graph to the relevant lines in your code.
 - Find resource consumption broken down by method name and line number.
 
+The **Logs Navigation** support opens the Datadog Log Explorer with a view matching the context in which you are working.
```
[the remark this answers is not in the document]
Comment (jfree) [reply content/en/developers/ide_integrations/idea/_index.md:45]: The idea is for the Overview to mention each main feature the plugin offers.  Perhaps the real issue is that the description for the Continuous Profiler is too long? (previously it was the _main_ feature)

**Review by aliciascott — APPROVED** [review]
Looks good!

**Pull request merged**